### PR TITLE
Issue #494: Add flag to control gesture continuity

### PR DESCRIPTION
**Pull request by vladcipariu91**
[Issue](https://github.com/bumble-tech/appyx/issues/494)

## Description

Added `isContinuous` Boolean flag to `Gesture`. This is set to `true` by default. However in cases where a new gesture being created with the remaining offset isn't desirable this can be set to false.

## Check list

- [ ] I have updated `CHANGELOG.md` if required.
- [ ] I have updated documentation if required.

### CODE REVIEW

File: appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/ui/gesture/Gesture.kt [hunk 1]
```diff
@@ -6,16 +6,24 @@ import com.bumble.appyx.interactions.core.ui.math.proportionOf
 
 open class Gesture<InteractionTarget, ModelState> internal constructor(
     val operation: Operation<ModelState>,
+    val isContinuous: Boolean,
```
Comment (zsoltk) [inline appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/ui/gesture/Gesture.kt]: What do you think if we moved it to `GestureFactory`? That way it wouldn't need to be specified per every gesture. The cases we have so far are either all of them are continuous or all of them are not.

Comment (vladcipariu91) [reply appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/ui/gesture/Gesture.kt]: the current option gives more control, however your suggestion is simpler to use, I would choose the existing one, but it's your call

Comment (zsoltk) [reply appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/ui/gesture/Gesture.kt]: Can be a combination:
- `GestureFactory` provides the default
- `Gesture` falls back to factory default but allows overriding if you want

Comment (zsoltk) [reply appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/ui/gesture/Gesture.kt]: I'm assuming based on the cases so far that it's either/or for the whole of it, so if it's only one choice I'd go with factory-level definition for its ease.

Comment (vladcipariu91) [reply appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/ui/gesture/Gesture.kt]: i'll update to only use on the factory

Comment (vladcipariu91) [reply appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/ui/gesture/Gesture.kt]: @zsoltk done!

**Review by zsoltk — APPROVED** [review]

**Pull request merged**